.github/workflows/smoke.yml

```diff
@@ -32,7 +32,12 @@ jobs:
 
       # Download the Proxy cache. The job is ideally 100% cached so no real calls are made.
       - name: Download artifacts
-        run: script/download-cache.sh ${{ matrix.suite }}
+        run: |
+          script/download-cache.sh ${{ matrix.suite }}
+          # If the previous command failed, and this is a scheduled run, fail the job.
+          if [ $? -ne 0 ] && [ "${{ github.event_name }}" == "schedule" ]; then
+              exit 1
+          fi
```

Comment:
How would that work since `${{ success() }}` returns true when previous steps succeed, that means the previous step would have to fail which means the workflow is going to fail? We'd need an `if: ${{ always }}` or use `continue-on-error`. It seems simpler to stick with bash's conditional.

Reply:
🤦 I should have written `${{ failure() }}`. I was thinking something like [this example](https://docs.github.com/en/actions/learn-github-actions/expressions#example-of-failure-with-conditions)